ds3-autogen-api/src/main/java/com/spectralogic/ds3autogen/api/models/Ds3Type.java

```diff
@@ -101,4 +104,8 @@ public boolean equals(final Object obj) {
     public static boolean isEmpty(final Collection<?> collection) {
```

Comment:
I know this function wasn't just added, but isn't there another place where we have this method?